config/index.js

```diff
@@ -98,6 +98,7 @@ module.exports = {
     SHORT: 'd M yyyy',
     LONG: 'd MMM yyyy',
     WITH_DAY: 'EEEE d MMM yyyy',
+    WITH_DAY_AND_TIME: 'EEEE d MMM yyyy, h:mm:ss aa',
```

Comment:
I wonder if the seconds are ever really needed in this case? 

The format we've used elsewhere within the service when there is a time and date has been `(at) 3:30pm on Wed 10 Oct 2020`. This is a little more aligned with the [GOV.UK content guidelines](https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style#times) too.

But the addition here is a good one, and maybe a format from config should be used to ensure that consistency.

Reply:
Agree - match the format we have globally

Reply:
I've changed this to match the "time at date" format, as seen in the screenshot below